pkg/pillar/cmd/zedagent/reportinfo.go

```diff
@@ -45,7 +45,7 @@ var (
 	maxSmartCtlSize = 65536 // Limit size of smartctl output files
 )
 
-func deviceInfoTask(ctxPtr *zedagentContext, triggerDeviceInfo <-chan struct{}) {
+func deviceInfoTask(ctxPtr *zedagentContext, triggerDeviceInfo <-chan infoDest) {
```

Comment:
I'm consuded by having infoDest be a type in some functions like this one, and a variable name in other functions. Can you disambiguate the names?